packages/starlight/components/SidebarSublist.astro

```diff
@@ -87,7 +87,8 @@ interface Props {
 		border-radius: 0.25rem;
 		text-decoration: none;
 		color: var(--sl-color-gray-2);
-		padding: 0.1em var(--sl-sidebar-item-padding-inline);
+		padding: 0.3em var(--sl-sidebar-item-padding-inline);
+		line-height: 1.4em;
```

Comment:
Nit but we generally use unit-less line-height values (has the same effect anyway):

```suggestion
		line-height: 1.4;
```